suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -98,6 +98,8 @@ export const SendOutputsScreen = ({
         selectSendFormDraftByAccountKey(state, accountKey),
     );
 
+    const network = account ? getNetworkCompatible(account.symbol) : null;
```

Comment:
please, change not to use deprecated method:

```
/**
 * @deprecated use `networks[symbol]` (exact type) or `getNetwork` (generic type) from '@suite-common/wallet-config'
 */
```

Reply:
Could we please leave it as it is? 🙏 @Lemonexe is already changing this file in his follow-up refactor PR: https://github.com/trezor/trezor-suite/pull/14398#discussion_r1771440838. Then the typing will behave correctly.

Reply:
Yes, the `composeContext` still works with `networkCompatible`, and that is refactored in the last PR.

I left it as the last thing, because it's heavily interconnected with `selectedAccountState`, and with suite discovery, so it'be a nightmare to refactor one without the other.

Reply:
okay, then 👍